src/sql/base/browser/ui/table/tableDataView.ts

```diff
@@ -14,7 +14,7 @@ export interface IFindPosition {
 	row: number;
 }
 
-function defaultSort<T>(args: Slick.OnSortEventArgs<T>, data: Array<T>): Array<T> {
+export function defaultSort<T>(args: Slick.OnSortEventArgs<T>, data: Array<T>): Array<T> {
```

Comment:
if this is going to be exported can you rename it something more specific, like `defaultDataSort` or something.